tests/tests.mk

```diff
@@ -40,7 +40,7 @@ test.kube.presubmit: init | $(JUNIT_REPORT)
 
 test.kube.postsubmit: test.kube.presubmit
 
-test.kube.%: init | $(JUNIT_REPORT)
+test.kube.directory.%: init | $(JUNIT_REPORT)
```

Comment:
Shouldn't we just remove this rule completely, given that we're not working on, or maintaining, the tests in the release-1.6 branch. If any 1.6 test fails too often, we should disable it.

Reply:
@frankbu I will remove it all together. I was just trying to leave a little something to change to in case others might have still used the directory target.